<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="PaintLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0.6 rgb(160, 161, 179), stop:1 rgba(160, 161, 179, 100))</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>425</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>410</y>
     <width>155</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 1px solid #A0A0A0;
	border-radius: 3px; 
	padding-left: 5px;
	background-color: #F2F2F2; 
	color: #A0A0A0; 
	selection-background-color: #A0A0A0; 
	selection-color: #F2F2F2; 
	font-family: &quot;Microsoft YaHei&quot;; 
	font-size: 14pt; 
}</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>175</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>345</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="TextLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>425</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="btnRight" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>60</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="btnLeft" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>200</y>
     <width>60</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label_1</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>lineEdit</zorder>
  <zorder>btnRight</zorder>
  <zorder>btnLeft</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintLabel</class>
   <extends>QLabel</extends>
   <header>Mainbkg</header>
  </customwidget>
  <customwidget>
   <class>TextLabel</class>
   <extends>QLabel</extends>
   <header>Mainbkg</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
